templates/mmm/abstract.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'base.html' %}
 {% block content %}
-<h3>Historical and Projected Temperature + Precipitation Minimum, Mean, and Maximum</h3>
-<p>The endpoint here provides access to the temperature at surface in degrees Fahrenheit and total annual precipitation in inches for both historical and projected model data over the state of Alaska.</p>
+<h3>Historical and Projected Minimum, Mean, and Maximum</h3>
+<p>The endpoint shere provide access to the temperature at surface in degrees Celsius, total annual precipitation in mm, and degree days for both historical and projected model data over the state of Alaska.</p>
```

Comment:
Typo in 'shere'

Reply:
Fixed in 52554fe79ce961400415f1113fab7293b18b90ab.